site/blog/2024-06-26-datapackage-v2-release/README.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Data Package version 2.0 is out!
+date: 2024-06-26
+tags: ["news"]
+category: news
+image: /img/blog/DP-release.png
+description: We are very excited to announce the release of Data Package v2
+author: Sara Petti
+---
+We are very excited to announce the release of the version 2.0 of the Frictionless Standard. Thanks to the generous support of [NLnet](https://nlnet.nl/) from November last year we were able to [focus on reviewing the Frictionless Standard](https://frictionlessdata.io/blog/2023/11/15/frictionless-specs-update/#additional-deliverables) in order to include features that were often requested throughout the years and improve extensibility for domain-specific implementations. 
```

Comment:
Not sure if now we need to mix this Fricitionless / Data Package Standard. Maybe we just need to mention that during the work, the new name was established to avoid confusion.

Reply:
Do you think the "We are very excited to announce the release of the version 2.0 of Data Package (previously known as Frictionless Specs)" would work better?

Reply:
I agree, something like this